osu.Game/Online/Leaderboards/Leaderboard.cs

```diff
@@ -166,6 +161,7 @@ protected PlaceholderState PlaceholderState
         }
 
         protected Leaderboard()
+            : base("view online leaderboards!")
         {
             InternalChildren = new Drawable[]
```

Comment:
Have you tested this correctly, it seems that all internal children of the `OnlineViewContainer` got ripped out.